src/logcapture.cpp

```diff
@@ -60,9 +66,8 @@ LogCapture::LogCapture(const char *file, const int line, const char *function, c
 * See also for the attribute formatting ref:  http://www.codemaestro.com/reviews/18
 */
 void LogCapture::capturef(const char *printf_like_message, ...) {
-   static const int kMaxMessageSize = 2048;
    static const std::string kTruncatedWarningText = "[...truncated...]";
-   char finished_message[kMaxMessageSize];
+   char finished_message[MaxMessageSize];
```

Comment:
It is illegal c++ to create a stack c-array with a dynamic size. You can get around this by instead using a std::vector as the container and get the c-array from std::vector::data()